digital_land/package/organisation.py

```diff
@@ -0,0 +1,266 @@
+import os
+import csv
+import sys
+import json
+import requests
+import logging
+from pathlib import Path
+
+from .csv import CsvPackage
+
+DATASET_URL = "https://files.planning.data.gov.uk/organisation-collection/dataset/"
```

Comment:
Should we define it here? what do you think about putting it in a makefile or some centralized place?